lib/daisy_ui_components/tabs.ex

```diff
@@ -0,0 +1,250 @@
+defmodule DaisyUIComponents.Tabs do
+  @moduledoc """
+  Tab component
+
+  Renders a list of tab links or toggles with radio toggle and content or label with radio toggle and content
+
+  https://daisyui.com/components/tab/
+  """
+
+  use DaisyUIComponents, :component
+
+  @doc """
+  Renders a Tabs with links as toggle by default.
+
+  ## Default Example
+  <.tabs>
```

Comment:
how about we use like this:
```heex
  <.tabs>
    <:tab title="Tab 1">Tab Content</:tab>
    <:tab  title="Tab 2" active>Tab 2 Content</:tab>
    <:tab title="Tab 3" disabled />
  </.tabs>
```
This way we wouldn't need the tab content slot, since the inner_block would be the content.
`tab_link` could (for example) take the `title` attribute to add under the <a> element

Reply:
I like this. Its nice and simple, but I think the tab needs to be able to take more than just a string title. From the docs there is an example where it gets rendered with icons, so I think it should take an inner block. If we limit it to just title, icon and icon position attribute, having the tab content inside the `<:tab>` slot would simplify things quite a bit.